docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -1999,6 +1999,41 @@ use custom implementations of the following components:
 When creating custom implementations of Elytron components, they must
 present the appropriate capabilities and requirements.
 
+=== Configuring SNI
+
+Using the WildFly Elytron subsystem it is possible to configure an SSL context which supports SNI.  By supporting SNI if an SNI host name is available whilst the SSLSession is being negotiation a host specific SSLcontext will be selected.  If no host specific SSLContext is identified either because no host name was received or because there is no match a default SSLContext will be used instead.  By identifying a host specific SSLContext it means that a certificate appropriate for that host can be used.
```

Comment:
Minor detail but
> host specific SSLcontext

Should probably be `SSLContext` to match the rest of the case.

Reply:
also in that same clause...

s/is being negotation/is being negotiated/g